internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -866,6 +867,127 @@ func TestAccClusterAdvancedCluster_priorityNewSchema(t *testing.T) {
 	})
 }
 
+func TestAccAdvancedCluster_oldToNewSchemaWithAutoscalingDisabled(t *testing.T) {
```

Comment:
Would consider cover this case with existing shardedMigrationFromOldToNewSchema test case (which does not define autoscaling)